### PR TITLE
Adds the rusted spigot

### PR DESCRIPTION
**Pull request by alexkar598**
I was bored and mapped the shuttle that Chronarch(BEER) had on paper,here it is!(admin spawnable only)

test

To make this possible i made the emergency shuttle docks a bit bigger and removed some trees at CC nukie dock(where they would be invisible unless you got and looked by the shuttle window insteed of being super happy about your hijack)

### CODE REVIEW

**Comment by alexkar598** [comment]
travis shitted on me

**Comment by Plotron** [comment]
So much wood...

**Comment by taukausanake** [comment]
Can we get some before and afters of the maps you edited just in case?

**Comment by alexkar598** [comment]
Which map?

**Comment by Fox-McCloud** [comment]
Don't we already have a bar shuttle. I seem to recall we have one, which makes this a bit redundant.

**Comment by alexkar598** [comment]
Both seem different
bar
spigot

**Comment by Spacemanspark** [comment]
I like it, but perhaps it could use a bit more aesthetic. Some plants, perhaps?

**Comment by alexkar598** [comment]
Dunno,ask @Chornarch2954

**Comment by alexkar598** [comment]
Plants and space piano were added @Spacemanspark

**Comment by Kyep** [comment]
Does changing the shuttle dimensions affect metastation, and other maps?
I ask because I notice you edited z2, but not meta z2.

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -51,6 +51,11 @@
 	description = "Troop transport with point defense turrets."
 	admin_notes = "Designed to ensure a safe evacuation during xeno outbreaks."
 
+/datum/map_template/shuttle/emergency/spigot
+	suffix = "spigot"
+	name = "The Rusted Spigot"
+	description = "A barship owned by BEER."
+	admin_notes = "Has a basic medical facility,a lot of booze(might be worth spawning some antihol pills) and a cargo hold.Shuttle is not recommended for chaotic rounds."
```
Comment (Kyep) [inline code/datums/shuttles.dm]: Spacing.

**Comment by alexkar598** [comment]
uhh...right lemme get pics for z2 and update meta aswell

**Comment by alexkar598** [comment]
image
cyberiad z2 nukie dock

**Comment by Citinited** [comment]
Here are my thoughts on this:

- Bartender needs a seat on the other side of the bar so he doesn't fall over
- There aren't many emergency O2 lockers on this shuttle - intentional?
- I think you need to make those wooden turfs and plating shuttle turfs, otherwise the shuttle is susceptible to a bunch of stuff
- That extra (off-centre) windoor in front of the command airlock is probably going to be supremely irritating for anyone trying to emag their way into the cockpit. Remove it? 
- Maybe a second airlock to the east of the security cell to easily move cockpit assailants into brig?
- More props would be nice, maybe a deck of cards or something
- That fueltank looks to serve no purpose except to explode and get in the way

**Comment by Spacemanspark** [comment]
You can flip the tables and immediately negate the windoor issue

**Comment by taukausanake** [comment]
You can flip that many tables? I thought the limit would be three or four, not five.

Maybe add a single table or two at top and move the bar stuff down a tile?

**Comment by Spacemanspark** [comment]
You can flip the tables in front of the command door and negate the windoor issue is what I should have said.

**Comment by alexkar598** [comment]
Intended,also DNM until I ensure this works for met-wait...do we support custom shuttles for meta?

**Comment by DarkPyrolord** [comment]
After a little bit of discussion it has been decided that this PR it to be closed, we already have a bar themed shuttle and we don't really need two bar themed shuttles, if you want to though, feel free to go redesign the bar shuttle a little, I'd be all good with that myself.

**Comment by alexkar598** [comment]
Could I add this shuttle as a ruin perhaps?

**Pull request closed without merging**